src/routes/routes.ts

```diff
@@ -110,3 +111,15 @@ defaultRoute.get('/catalogo_prodotti' ,get_prodotti, (req:Request,res:Response)=
 defaultRoute.get('/prodotto/acquisto/:id' ,tokenCheck, acquisto, (req:Request,res:Response)=>{
     res.json(req.body)
 })
+
+defaultRoute.post('/segnalazione/:id' ,tokenCheck, segnala, (req:Request,res:Response)=>{
```

Comment:
Costruire un router dedicato in cui definire le route e poi importarlo in routes.ts